src/plots/cartesian/graph_interact.js

```diff
@@ -60,7 +60,7 @@ fx.supplyLayoutDefaults = function(layoutIn, layoutOut, fullData) {
     coerce('dragmode');
 
     var hovermodeDflt;
-    if(layoutOut._hasCartesian) {
+    if(layoutOut._hasCartesian && !layoutOut._hasPie) {
```

Comment:
But this is not really the same logic... what if you have both pie and cartesian for real?

I know pie is my own doing, but why does it need to have cartesian as its `basePlotModule`? Can we rip that out into its own thing? That's probably a bigger project, but as it is it's not only hacky but tough to get the logic right.